lms/static/js/student_account/views/LoginView.js

```diff
@@ -86,6 +92,16 @@
 
             resetEmail: function() {
                 this.element.hide( this.$errors );
+                this.resetMessage = this.$resetSuccess.find('.message-copy');
+                if (this.resetMessage.find('p').length == 0){
```

Comment:
In case you didn't see it, the failure is listed here on the JSHint tab:

https://build.testeng.edx.org/job/edx-platform-quality-pr/22358/Diff_Quality_Report/